app/poi_api/poi_errors.py

```diff
@@ -2,10 +2,10 @@
 
 
 class POINotFoundError(BaseApiError):
-    def __init__(self, message, error_code):
+    def __init__(self, message, error_code=404):
         super().__init__(message, error_code)

```

Comment:
I think this pattern would work well if you added a `error` as an extra param that you set statically for each error to distinguish them. As it stands you could just replace them with the base class. E.g. `super("Poi could not be found", message, 404)`